app/controllers/tags_controller.rb

```diff
@@ -131,6 +131,7 @@ def feed
     end
   end
 
+  # TODO: Refactor the tag_type checking logic
```

Comment:
Was this something you needed to come back to?

Reply:
Yeah, this method is in need of some TLC. I added a few lines to it. I think I wanted to clean it up and refactor some here and there, which I can still do.

Reply:
I cleaned this up a little and removed the comment. This can be further refactored, but will take a considerable more effort.